rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search/240_date_nanos.yml

```diff
@@ -91,8 +91,40 @@ setup:
           docvalue_fields:
             - field: date
```

Comment:
I consider fetching DV fields multiple times with different formats a misuse, and we don't need to keep BWC for this.

Reply:
👍 makes sense to me too